src/Clapi/Relay.hs

```diff
@@ -124,27 +125,28 @@ relay vs = waitThenFwdOnly fwd
             nsContOp (StructDef (StructDefinition _ kids)) = Map.singleton ns
               (Nothing, SoPresentAfter $ presentAfter ns $ alKeys kids)
             nsContOp _ = error "Root def not a struct WTAF"
-            defs' = vsMinimiseDefinitions defs vs
-            dd' = vsMinimiseDataDigest dd vs
-            contOps' = vsMinimiseReords contOps vs
+            qDd = maybe (error "Bad sneakers") id $ alMapKeys (ns :</) dd
```

Comment:
"Bad sneakers"? Am I missing a reference?!

Reply:
It's a Steely Dan song, and I don't think it is an error I've used before.